module2-avro/src/main/java/com/mentoring/avro/service/OrderConsumerService.java

```diff
@@ -0,0 +1,20 @@
+package com.mentoring.avro.service;
+
+import com.mentoring.avro.model.Order;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.support.Acknowledgment;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.stereotype.Service;
+
+@Service("NotificationService")
```

Comment:
Do you really need a different java class name and spring bean name?
If it's **notificationService** then the same name should be applied for java class name